packages/components/Avatar/SPEC.md

```diff
@@ -203,6 +205,16 @@ export interface AvatarTokens extends IBackgroundColorTokens, IForegroundColorTo
    */
   badgeSize?: BadgeSize;
 
+  /**
+   * The x position of presence badge.
+   */
```

Comment:
Can you elaborate position relative to which element as [0,0].

Reply:
Updated.